### DISCUSSION TITLE
Adding a "Download Report" button for Lecturers to download data at the end of a class

### DISCUSSION THREAD

**Discussion by Redghoul1234** [ashongdev/record-attendance#22] [Ideas]
Adding a "Download Report" button for Lecturers to download data at the end of a class

**Comment by ashongdev** [comment]
So we were thinking of adding a button where lecturers can download a summary of attendance reports? Like being able to download PDF file at the end of each class? 

What do you think?

**Reply by Bishop105** [reply]
Great

**Reply by Caleb457** [reply]
Nice 👍

**Reply by Abenadk** [reply]
Good idea

**Reply by akosuaDama** [reply]
> So we were thinking of adding a button where lecturers can download a summary of attendance reports? Like being able to download PDF file at the end of each class?
> 
> What do you think?

Good idea

**Comment by Redghoul1234** [comment]
not bad but it would be a hectic task though

**Comment by Bishop105** [comment]
Lets see the UI again

**Reply by akosuaDama** [reply]
Yeah let’s the UI

**Comment by Caleb457** [comment]
Okay

**Reply by JusticeAflade** [reply]
okay

**Comment by Redghoul1234** [comment]
![Screenshot (46)](https://github.com/user-attachments/assets/28a61d2d-2e44-4a85-8b69-ba76b6eac9e3)
look at where the student list button is
we vcould put the download button under it and center it as well

**Reply by Caleb457** [reply]
Yeah that will cool

**Reply by ashongdev** [reply]
> ![Screenshot (46)](https://private-user-images.githubusercontent.com/164070819/391132335-28a61d2d-2e44-4a85-8b69-ba76b6eac9e3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4OTU2MzYsIm5iZiI6MTczMjg5NTMzNiwicGF0aCI6Ii8xNjQwNzA4MTkvMzkxMTMyMzM1LTI4YTYxZDJkLTJlNDQtNGE4NS04YjY5LWJhNzZiNmVhYzllMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQxNTQ4NTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wODRiMzQ2YzIzMjM3OGNjZTlmMjc2N2E0OWMxOThiODU2NzlhZGIwYzJjY2MzNmYxZjVjOTgwMTFjOWE3ZWFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.zsfiw-3OKF_i7ZfcABdzDI1CCktQ66jRz57U6u7Weiw) look at where the student list button is we vcould put the download button under it and center it as well

I was thinking it should be on the page where the student list is being displayed. Like this
<img width="545" alt="Screenshot 2024-11-29 155010" src="https://github.com/user-attachments/assets/fba3debe-cd0f-4cbd-9c7b-b4a975d3e543">

PS: I've also updated the UI

**Reply by Redghoul1234** [reply]
so where under the course name or something

**Reply by ashongdev** [reply]
No. Its the print button Ive added beside the refresh button.
<img width="545" alt="Screenshot 2024-11-29 155010" src="https://github.com/user-attachments/assets/0e5c410b-06c4-4117-a462-59adf8a330e0">

**Reply by Redghoul1234** [reply]
does it work?

**Reply by Caleb457** [reply]
Do we need to add the refresh sign or this one is okay

**Reply by ashongdev** [reply]
Yeah it works

**Reply by Caleb457** [reply]
Nice👍

**Comment by Bishop105** [comment]
Check something pls, see the student side, there should be to show after you have successfully login

**Reply by Redghoul1234** [reply]
That's where the check in comes in

**Reply by Caleb457** [reply]
Yeah the check in

**Reply by Bishop105** [reply]
Ohk I see

**Comment by JusticeAflade** [comment]
okay
